Utils/handlerutil2.py

```diff
@@ -345,9 +348,14 @@ def do_heartbeat_report(self, heartbeat_file, status, code, message):
         if ext_utils.set_file_contents(heartbeat_file, health_report) is None:
             self.error('Unable to wite heartbeat info to ' + heartbeat_file)
 
-    def do_exit(self, exit_code, operation, status, code, message):
+    def do_exit(self, exit_code, operation, status, code, message,error_code=None):
         try:
-            self.do_status_report(operation, status, code, message)
+            if error_code is not None:
+                substatus = [{
+                    "Name": "ErrorClarification",
```

Comment:
NIT - The word "ErrorClassification" should be put in a place where constants in the repo are kept. Generally speaking it is a good idea to have all magic strings, magic numbers etc. put in dedicated files and reference the constants through that instead of putting the literals in everywhere in code.